### ISSUE TITLE
Add support for upto 31 band equalizer

### ISSUE THREAD

**Issue by max5354** [Pittvandewitt/Wavelet#190]
It will be great if you add more than 10 band equalizer like 31 band equalizer. Overall it will increase audio quality.

**Comment by ectv96** [comment]
agree! we need more bands and wider dB limit (15dB ups and 15dB downs)

and will be so much better if u guys add PARAMETRIC EQ too!
